ApplyToBecomeInternal/ApplyToBecomeInternal/Views/Shared/_SubMenu.cshtml

```diff
@@ -8,7 +8,7 @@
 		</li>
 
 		<li class="moj-sub-navigation__item">
-			<a class="moj-sub-navigation__link" @(Model.Page == SubMenuPage.ApplicationForm? "aria-current=page" : string.Empty) href="/application-form/@Model.ProjectId">Application form</a>
+			<a class="moj-sub-navigation__link" @(Model.Page == SubMenuPage.ApplicationForm? "aria-current=page" : string.Empty) href="/school-application-form/@Model.ProjectId">School application form</a>
```

Comment:
Could you update it to use the anchor tag helper? https://docs.microsoft.com/en-us/aspnet/core/mvc/views/tag-helpers/built-in/anchor-tag-helper?view=aspnetcore-3.1